corehq/apps/reports/tasks.py

```diff
@@ -46,6 +44,12 @@
 from corehq.util.files import file_extention_from_filename
 from corehq.util.view_utils import absolute_reverse
 
+from .analytics.couchaccessors import (
+    get_form_ids_having_multimedia as couch_get_form_ids_having_multimedia
+)
+from .analytics.esaccessors import (
+    get_form_ids_having_multimedia as es_get_form_ids_having_multimedia
```

Comment:
Could just name the functions differently which would also prevent anyone importing the wrong one by accident.